client/src/test/java/org/apache/rocketmq/client/consumer/DefaultMQPushConsumerTest.java

```diff
@@ -52,6 +56,7 @@
 import org.apache.rocketmq.common.protocol.header.PullMessageRequestHeader;
 import org.apache.rocketmq.remoting.exception.RemotingException;
 import org.junit.After;
+import org.junit.Assert;
```

Comment:
Hi, let's unify the assert tool and use `org.assertj.core.api.Assertions.assertThat`.